### PR TITLE
Increasing the requested resource cpu

### PR DESCRIPTION
**Pull request by prietyc123**
Increasing the resource to make sure it is not one of the reason for deployment rollout as suspected in issue https://github.com/openshift/odo/issues/3256

ping @amitkrout

### CODE REVIEW

File: ci-operator/config/openshift/odo/openshift-odo-master__v4.5.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ tests:
       from: oc-bin-image
       resources:
         requests:
-          cpu: "2"
```
Comment (girishramnani) [inline ci-operator/config/openshift/odo/openshift-odo-master__v4.5.yaml:37]: Isn’t this reducing the resources? From 2 to 800m? 800m means millicpus so effectively .8. Can we not reduce 2 to 800m?

Comment (girishramnani) [reply ci-operator/config/openshift/odo/openshift-odo-master__v4.5.yaml:37]: This shows that the periodic job gets 2 cpus but the CI just gets 200m, so we can keep the 2 cpu in periodic job or increase that? But definitely increase the 200m in standard jobs

Comment (prietyc123) [reply ci-operator/config/openshift/odo/openshift-odo-master__v4.5.yaml:37]: > Isn’t this reducing the resources? From 2 to 800m? 800m means millicpus so effectively .8. Can we not reduce 2 to 800m?

@girishramnani So as per our discussion on slack `800m` is enough cpu limit for our test to run, right? Isn't it be ok if we reduce the resources according to our consumption.

Comment (girishramnani) [reply ci-operator/config/openshift/odo/openshift-odo-master__v4.5.yaml:37]: I don’t want to unnecessarily change what’s already set. So let’s increase the 200m to 800m and keep the 2 cpu as is

File: ci-operator/config/openshift/odo/openshift-odo-master__v4.4.yaml [hunk 2]
```diff
@@ -35,7 +35,7 @@ tests:
       from: oc-bin-image
       resources:
         requests:
-          cpu: "2"
+          cpu: 800m
           memory: 4Gi
```
Comment (amitkrout) [inline ci-operator/config/openshift/odo/openshift-odo-master__v4.4.yaml:39]: May be you can increase the memory to 6Gi to avoid pod rollout issue

**Review by amitkrout — CHANGES_REQUESTED** [review]

**Comment by prietyc123** [comment]
As per the commit https://github.com/openshift/release/pull/11695/commits/b7c2ffb1b60c4f1ad5df2f883c90396b3b705377

/cancel hold

**Pull request merged**